src/services/doppler-api-client.double.ts

```diff
@@ -62,14 +62,14 @@ const campaignDeliveryCollection = {
       campaignId: 2,
       campaignName: 'Campaña calendario estacional 2019',
       campaignSubject: 'El calendario estacional 2019 ya está aquí',
-      deliveryStatus: 'opened',
+      deliveryStatus: 'notOpened',
       clicksCount: 23,
     },
     {
       campaignId: 3,
       campaignName: 'Emms 2019 preveento 1',
       campaignSubject: 'Ya comienza el dia 2. Accede a las conferencias',
-      deliveryStatus: 'opened',
+      deliveryStatus: 'softBounced',
```

Comment:
Perhaps you could add all cases for testing right?

Reply:
i added now